src/js/edu-benefits/1990e/helpers.jsx

```diff
@@ -8,3 +10,24 @@ export function transform(formConfig, form) {
     }
   });
 }
+
+export function eligibilityDescription() {
+  return (
+    <div className="usa-alert usa-alert-warning usa-content">
+      <div className="usa-alert-body">
+        <ul>
+          <li>You may be eligible for more than 1 education benefit program.</li>
+          <li>You can only get payments from 1 program at a time.</li>
+          <li>You can’t get more than 48 months of benefits under any combination of VA education programs.</li>
+        </ul>
+      </div>
+    </div>
+  );
+}
+
+export const benefitsLabels = {
+  chapter33: <p>Post-9/11 GI Bill (Chapter 33)<br/><a href="/education/gi-bill/post-9-11/" target="_blank">Learn more</a></p>,
```

Comment:
Are these the same as the 1995? Maybe we should put them in a helpers file in edu-benefits/utils that both forms could share?